app/src/main/java/com/jerboa/api/Http.kt

```diff
@@ -121,7 +121,7 @@ object API {
         }
     }
 
-    suspend fun createTempInstance(
+    private suspend fun createTempInstance(
```

Comment:
Needs to stay public, not used rn publicly but can in the future, no gain by restricting it

Reply:
Understand